### PR TITLE
docs: Fix Typos Across Various Documentation Files

### PR DESCRIPTION
**Pull request by joaolago1113**
This pull request addresses multiple minor typographical errors found in the documentation. The corrections are listed below for each commit:

1. *Commit 4f416a5c87bf*: Corrected 'Additionaly' to 'Additionally' in docs/operators/how-to/configure-store.md.

2. *Commit b68b3d58dcbd*: Removed a redundant 'to' in the configuration instructions of docs/operators/how-to/configure-store.md.

3. *Commit 82ff94651ded*: Changed 'usin' to 'using' in docs/operators/how-to/configure.md.

4. *Commit 6f3b81b1bc34*: Corrected 'mechanims' to 'mechanisms' in docs/operators/how-to/configure.md.

5. *Commit 67cd6c4e3bde*: Fixed typo 'samp' to 'spam' in the console message description in docs/tutorial/rln-chat2-live-testnet.md.

6. *Commit 75e591b531d9*: Added missing 'of' in the job description of docs/contributors/continuous-integration.md.

7. *Commit 10c2e334066e*: Fixed a duplicate 'to' in the Release Process section of docs/contributors/release-process.md and docs/operators/README.md.

8. *Commit 8617632adcb1*: Changed 'publically' to 'publicly' in waku/README.md.

These changes aim to improve the clarity and professionalism of the documentation. Each fix is small but contributes to the overall quality and readability of the project's documentation.

### CODE REVIEW

**Review by Ivansete-status — APPROVED** [review]
Lovely! Thank you so much for it :100: !
You'll be able to merge the PR once you get two approvals, that will happen soon :)

**Review by gabrielmer — APPROVED** [review]
Thanks so much!

**Review by SionoiS — APPROVED** [review]

**Review by NagyZoltanPeter — APPROVED** [review]
Thank you for taking care!

**Pull request merged**
